### PR TITLE
Fixes and tweaks based on feedback

### PR DESCRIPTION
**Pull request by hcarneiro**
List of feedback here: https://docs.google.com/spreadsheets/d/1iyGJHw5Yrol-zttFfHbue1NxQGJ4pJVAsHunHOlpXKM/

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <!-- Nav tabs -->
     <ul class="nav nav-tabs" role="tablist">
       <li class="active"><a href="#notification-send-tab" data-toggle="tab">Send notifications</a></li>
-      <li><a href="#report" data-toggle="tab" id="note-reports">Reports</a></li>
-      <li><a href="#settings" data-toggle="tab">Push interface</a></li>
+      <li><a href="#report" data-toggle="tab" id="note-reports">Sent notifications &amp; Reports</a></li>
```
Comment (tonytlwu) [inline interface.html]: How is **Sent notifications** different from **Reports**? If not then just go with one.

File: interface.html [hunk 2]
```diff
@@ -155,57 +155,59 @@ <h4 class="text-warning text-center">Your are about to send notifications to <st
...
-          </div>
-        </div>
-        <div class="form-group clearfix">
-          <div class="col-sm-12 control-label">
-            <label for="popup_message">Notification popup message</label>
-          </div>
-          <div class="col-sm-12">
-            <textarea class="form-control" name="popup_message" id="popup_message" rows="4" placeholder="Enter a message">{{#if popupMessage}}{{popupMessage}}{{else}}You will receive notifications when app updates are available.
-Please tap on Allow Notifications below to receive notifications on the updates.{{/if}}</textarea>
+        <p>Configure the popup asking users for permission to receive push notifications.</p>
```
Comment (tonytlwu) [inline interface.html]: Does this work?

_To receive notifications, your app users will be asked for permission. Configure the popup message below:_

File: interface.html [hunk 3]
```diff
@@ -250,15 +255,15 @@ <h4 class="text-warning text-center">Your are about to send notifications to <st
     <div class="col-xs-12">
       <div class="report-other-details">
         <div class="report-recipients">
-          <p>N<sup>o</sup> Recipients (iOS/Android/Win10)</p>
+          <p>N<sup>o</sup> Recipients (iOS/Android/Win10) <span class="question-icon" data-toggle="tooltip" data-placement="auto bottom" title="The number of users that accepted to receive push notifications from the app."><i class="fa fa-question-circle"></i></span></p>
```
Comment (tonytlwu) [inline interface.html]: _Number of devices that registered top receive push notification from the app._

File: interface.html [hunk 4]
```diff
@@ -250,15 +255,15 @@ <h4 class="text-warning text-center">Your are about to send notifications to <st
...
           <p class="report-recipients-data">\{{recipientsCount}} (\{{sentApple}}/\{{sentGoogle}}/\{{sentWindows}})</p>
         </div>
         <div class="report-delivery">
-          <p>Accepted for delivery</p>
+          <p>Accepted for delivery <span class="question-icon" data-toggle="tooltip" data-placement="auto bottom" title="Fliplet sends a request to Apple/Google/Microsoft to deliver the push notifications to the devices. This shows how many were accepted. However, whether they do or not accept is out of Fliplet's control."><i class="fa fa-question-circle"></i></span></p>
```
Comment (tonytlwu) [inline interface.html]: _Number of push notification requests accepted by Apple/Google/Microsoft to deliver the devices. Push notification requests may be rejected if the device registered is no longer valid._

File: interface.html [hunk 5]
```diff
@@ -250,15 +255,15 @@ <h4 class="text-warning text-center">Your are about to send notifications to <st
...
           <p class="report-delivery-data">\{{deliveredCount}} (\{{deliveryPerct}}%)</p>
         </div>
         <div class="report-batch">
-          <p>Batches (Sent/Total)</p>
+          <p>Batches (Sent/Total) <span class="question-icon" data-toggle="tooltip" data-placement="auto left" title="The number of push notification batches that were sent."><i class="fa fa-question-circle"></i></span></p>
```
Comment (tonytlwu) [inline interface.html]: _Fliplet sends the push notifications to Apple/Google/Microsoft in batches. This shows the number of batches that have been processed and sent._

Comment (squallstar) [reply interface.html]: True.

File: interface.html (hunk 4 above)
Comment (squallstar) [reply interface.html]: Correct

File: interface.html (hunk 3 above)
Comment (squallstar) [reply interface.html]: Correct.

**Pull request merged**